at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -267,6 +267,19 @@ class AtLookupImpl implements AtLookUp {
     }
     // If response starts with error:, throw AtLookupException.
     if (_isError(verbResult)) {
+      // If server returns the error response in JSON Encoded string, decodes the JSON String
+      // and throws the appropriate AtException basing on the error code.
+      if (verbResult.contains('errorCode')) {
```

Comment:
contains check may fail if some random data from server has errorCode in it..
error: <text> - old format
error: <json> - new format
_isError(verbResult)) can have the previous implementation..if json parsing fails, then consider error response to be in old format

Reply:
Modified the code to have error: before JSON response.